data/scenarios/tutorial.sav

```diff
@@ -244,19 +247,19 @@ function tutorial_city_growth_callback(city, change, reason)
+_("First size 5 city: Now your city has grown to size five. As cities get larger unrest becomes a problem. A city of this size will usually have one unhappy citizen unless pacifying effects are used; meaning one entertainer specialist is required to keep your citizens content. This is basically a wasted citizen. There are several things that can be done about this. One quick fix is to build a Temple [or other cultural building] that will make an unhappy citizen content. See the help on City Improvements for specs on each building type. As you get more large cities, it may pay off to change your tax rates [by pressing ctrl-T or clicking the button on the top bar] to dedicate some of your taxes directly to luxury goods. Every two luxury goods will pacify one citizen. Building a Marketplace enhances the benefit from this by providing +50 percent luxuries to the city. Pro tip: In the city dialog, you can click on the city center and the game will automatically arrange the citizens to manage the land with max food in mind. It will also keep an entertainer if needed."))
       growth5msg = true
     elseif size == 8 and not growth8msg then
       notify.event(city.owner, city.tile, E.SCRIPT,
-_("First size 8 city: Finally you have grown a city to size eight. A size eight city can provide substantial output provided you have enough luxuries to keep your population content. To grow a city beyond size eight requires an Aqueduct. If your city is going to continue to grow you should start building one soon. Building an Aqueduct requires the Construction technology, unless your city is next to lake or river. Pro tip: You should have a look at the small and great wonders. Small wonders are large buildings that give a bigger effect than a regular building and often work across all of your cities and citizens. Every player can build small wonders. Great wonders are game unique buildings that offer one player to have a big possitive effect across his nation. Great wonders can be expensive to build, but are often worth the effort."))
+_("First size 8 city: Finally you have grown a city to size eight. A size eight city can provide substantial output provided you have enough luxury goods to keep your population content. To grow a city beyond size eight requires an Aqueduct. If your city is going to continue to grow you should start building one soon. Building an Aqueduct requires the Construction technology, unless your city is next to lake or river. Pro tip: You should have a look at the small and great wonders. Small wonders are large buildings that give a bigger effect than a regular building and often work across all of your cities and citizens. Every player can build small wonders. Great wonders are game unique buildings that offer one player to have a big possitive effect across his nation. Great wonders can be expensive to build, but are often worth the effort."))
```

Comment:
Typo: missing "a" before "lake" and "river" in "unless your city is next to lake or river"